src/extension.ts

```diff
@@ -90,19 +91,19 @@ async function registerSkillActionComponents(context: vscode.ExtensionContext):
 
         registerWebviews(deploySkill, simulateSkill, syncInteractionModelView, aplPreviewWebView, syncManifestView);
 
-        ext.askSkillCommands = [new DeploySkillCommand(deploySkill), 
+        ext.askSkillCommands.push(new DeploySkillCommand(deploySkill), 
             new SyncInteractionModelCommand(syncInteractionModelView), new GetSkillIdFromWorkspaceCommand(), new SyncManifestCommand(syncManifestView),
             new CreateAplDocumentFromSampleCommand(aplPreviewWebView), new ChangeViewportProfileCommand(aplPreviewWebView), 
             new PreviewAplCommand(aplPreviewWebView), new SyncAplResourceCommand(),
             new RefreshSkillActionsCommand(), new SimulateSkillCommand(simulateSkill),
             new SimulateReplayCommand(simulateSkill), new ChangeSimulatorViewportCommand(simulateSkill)
-        ];
+        );
     
         apiRegisterCommands(context, ext.askSkillCommands);
 
         if (skillFolders.length === 1) {
-            // tslint:disable-next-line: no-unused-expression
-            new SkillActionsView(context);
+            const skillActionView = new SkillActionsView(context);
+            ext.treeViews.push(skillActionView);
```

Comment:
It's good idea to store the treeViews and webViews in this way, but it's breaking change and need to be careful. I tested and didn't find bugs. One point is the name of the namespace `ext`, it should be capitalized and more meaningful for easier search. Ref: https://www.typescriptlang.org/docs/handbook/namespaces.html

Reply:
Ahh, I don't think it's a breaking change since it won't influence the use case and user experience lol. 

For the naming `ext`, I just follow the trend of other extensions such as [aws toolkit](https://github.com/aws/aws-toolkit-vscode/blob/master/src/shared/extensionGlobals.ts#L18) and [docker toolkit](https://github.com/microsoft/vscode-docker/blob/master/src/extensionVariables.ts#L23). I am open to better suggestions :)